samcli/hook_packages/terraform/hooks/prepare/resource_linking.py

```diff
@@ -207,8 +207,15 @@ def _build_module_resources_from_configuration(module_configuration: Dict, modul
 
         expressions = config_resource.get("expressions", {})
         for expression_name, expression_value in expressions.items():
+            # we do not process the attributes of type dictionary
+            # Todo add dictionary type attributes post beta
+            if isinstance(expression_value, list):
+                LOG.debug("Skip processing the attribute %s as its value is a map.", expression_name)
```

Comment:
nit: we check for list and have debug message on "map"?

Reply:
This is how it works in Terraform, the customer defines an attribute and can set its value to a map, but in the Terraform state,  it represented as list of dictionaries.